tasks/pipeline-osdeps.yml

```diff
@@ -96,7 +96,7 @@
 # But yum or apt modules doesn't allow loops (deprecated) so the trick is using the
 # install-packages.yml task file
 
-- include_tasks: "install-packages.yml"
+- include_tasks: "install-packages.yml"  # We need to use include_tasks and not import_tashs because it contains dynamic data in loop
```

Comment:
```suggestion
- include_tasks: "install-packages.yml"  # We need to use include_tasks and not import_tasks because it contains dynamic data in loop
```

Reply:
Thanks! Fixed